_clients/javascript/index.md

```diff
@@ -32,8 +34,133 @@ Then require the client:
 ```javascript
 const { Client } = require("@opensearch-project/opensearch");
 ```
+{% include copy.html %}
 
-## Sample code
+## Connecting to OpenSearch
+
+To connect to the default OpenSearch host, create a client object with the address `https://localhost:9200` if you are using the Security plugin:  
```

Comment:
When you tack on "... if you are using the Security plugin." to the end, it implies there's an alternative way, or an alternative address to use, to create a client object when not using the Security plugin. Is there a distinction, and is it worth mentioning?

Reply:
If you're not using the security plugin, then the changes are trivial (you just need to use "http" instead of "https" and omit the certificates).

Reply:
It is addressed in the installation documentation. I can see if there is a need to address this here specifically, and if so I'll address this in another PR. I have a PR specifically for the javascript client.

Reply:
We should probably at least provide a link to information for those not using the Security plugin.